src/js/components/footer.js

```diff
@@ -1,39 +1,67 @@
 const behavior = require("../utils/behavior");
-const select = require("../utils/select");
 const { CLICK } = require("../events");
 const { prefix: PREFIX } = require("../config");
 
-const HIDDEN = "hidden";
 const SCOPE = `.${PREFIX}-footer--big`;
 const NAV = `${SCOPE} nav`;
 const BUTTON = `${NAV} .${PREFIX}-footer__primary-link`;
-const COLLAPSIBLE = `.${PREFIX}-footer__primary-content--collapsible`;
-
 const HIDE_MAX_WIDTH = 480;
 
+/**
+ * Expands selected footer menu panel, while collapsing others
+ */
 function showPanel() {
   if (window.innerWidth < HIDE_MAX_WIDTH) {
-    const collapseEl = this.closest(COLLAPSIBLE);
-    collapseEl.classList.toggle(HIDDEN);
-
-    // NB: this *should* always succeed because the button
-    // selector is scoped to ".{prefix}-footer-big nav"
-    const collapsibleEls = select(COLLAPSIBLE, collapseEl.closest(NAV));
+    const isOpen = this.getAttribute('aria-expanded') === 'true';
+    const thisFooter = this.closest(SCOPE);
 
-    collapsibleEls.forEach((el) => {
-      if (el !== collapseEl) {
-        el.classList.add(HIDDEN);
-      }
+    // Close all other menus
+    thisFooter.querySelectorAll(BUTTON).forEach((button) => {
+      button.setAttribute('aria-expanded', false);
     });
+
+    this.setAttribute('aria-expanded', !isOpen);
   }
 }
 
-const toggleHidden = (isHidden) =>
-  select(COLLAPSIBLE).forEach((list) =>
-    list.classList.toggle(HIDDEN, isHidden)
-  );
+/**
+ * Swaps the <h4> element for a <button> element (and vice-versa) and sets id
+ * of menu list
+ *
+ * @param {Boolean} isMobile - If the footer is in mobile configuration
+ */
+function toggleHtmlTag(isMobile) {
+  const footer = document.querySelector(SCOPE);
```

Comment:
This runs on all variants, let's limit it to the big footer variant only.

```suggestion
  const bigFooter = document.querySelector(SCOPE);

  if (!bigFooter) {
    return;
  }
```

Reply:
Good catch. This is done!